notebooks/community/gapic/custom/showcase_custom_image_classification_online_explain_example_based_api.ipynb

```diff
@@ -0,0 +1,3291 @@
+{
```

Comment:
<p>Markdown format issue with - __init__.py &nbsp;  - task.py ? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/716/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='86'/>

Reply:
<p>It is correct. We want <code>init.py</code>&nbsp; and  <code>task.py</code>   as files of <code>trainer</code> dir </p>